typing/typedtree.ml

```diff
@@ -844,3 +844,18 @@ let split_pattern pat =
         combine_opts (into cpat) exns1 exns2
   in
   split_pattern pat
+
+(* Expressions are considered short if they satisfy one of:
+
+   - Similar to an identifier: small words separated by '.' or '#'.
+   - Do not contain spaces when printed.
+   - Can be used as the subject of a sentence, eg. "'Foo.bar' is short." *)
```

Comment:
short ⇒ nominal . You could also restructure the comments to make this last condition the generally summary:

```
(* Expressions are considered nominal if they can be used as the subject of a sentence or action.In practice, we consider that an expression is nominal  if they satisfy one of:
   - Similar to an identifier: small words separated by '.' or '#'.
   - Do not contain spaces when printed.
  *)
```

Reply:
Updated, thanks.